proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -24,6 +24,11 @@ service Query {
     option (google.api.http).get = "/cosmos/auth/v1beta1/accounts/{address}";
   }
 
+  // AccountAddressByID returns account address based on account id 
+  rpc AccountAddressByID(QueryAccountAddressByIDRequest) returns (QueryAccountAddressByIDResponse) {
```

Comment:
`AccountAddressByID` seems like a weird name to me. Do people use the term "ID"?

I would call it `AddressByAccountNumber` or `AddressByAccNum`